R/FilterStateDate.R

```diff
@@ -1,17 +1,25 @@
+# DateFilterState ------
+
 #' @name DateFilterState
-#' @title `FilterState` object for Date variable
-#' @description Manages choosing a range of Dates
 #' @docType class
-#' @keywords internal
 #'
+#' @title `FilterState` object for Date variable
+#'
+#' @description Manages choosing a range of Dates.
 #'
 #' @examples
-#' filter_state <- teal.slice:::DateFilterState$new(
+#' # use non-exported function from teal.slice
+#' include_js_files <- getFromNamespace("include_js_files", "teal.slice")
+#' include_css_files <- getFromNamespace("include_css_files", "teal.slice")
+#' include_js_files <- getFromNamespace("include_js_files", "teal.slice")
```

Comment:
@kartikeyakirar why `include_js_files <- getFromNamespace("include_js_files", "teal.slice")` is repeated twice?

Reply:
thanks for catching this. I have removed it [88507b0](https://github.com/insightsengineering/teal.slice/pull/506/commits/88507b0bd36dd8f52df0895e23a9b403c16c660d)